README.md

```diff
@@ -33,7 +45,8 @@ Prerequisite: NPM and Node.js
   Examples:
 
     $ refocus-collector --help
-    $ refocus-collector start --collectorName=PRD_Collector_12345 --refocusUrl=https://refocus.foo.com --accessToken=eygduyguygijfdhkfjhkfdhg
+    $ refocus-collector start --collectorName=PRD_Collector_12345 --refocusUrl=https://refocus.foo.com --accessToken=eygduyguygijfdhkfjhkfdhg --refocusProxy=http://abcproxy.com --dataSourceProxy=http://xyzproxy.com
     $ refocus-collector stop --name=PRD_Collector_12345
     $ refocus-collector status --name=PRD_Collector_12345
     $ refocus-collector deregister --name=PRD_Collector_12345
+    $ refocus-collector reregister --collectorName=PRD_Collector_12345 --refocusUrl=https://refocus.foo.com --accessToken=eygduyguygijfdhkfjhkfdhg --refocusProxy=http://abcproxy.com
```

Comment:
I think all the commands which talk to Refocus (start, stop, status, deregister, reregister) should have optional refocusProxy arg... otherwise how will they talk to Refocus?